### PR TITLE
[DOC] Documentation for builtin OutlierIQR function

### PR DESCRIPTION
**Pull request by wizard-420**
This PR contains the documentation for outlierByIQR function

### CODE REVIEW

File: dev/docs/builtins-reference.md [hunk 1]
```diff
@@ -291,34 +291,62 @@ y = X %*% rand(rows=ncol(X), 1)
 [C, S] = steplm(X = X, y = y, icpt = 1);
 ```
 
-## `slicefinder`-Function
+## `outlier-Function
```
Comment (j143) [inline dev/docs/builtins-reference.md:295]: It seems `slicefinder` has been removed may be we should not.

**Comment by j143** [comment]
Now, in a new PR add the documentation for `outliersd`.

1. After which, you are aware of the outlier and it's flavours study a bit deeper and
2. And in the GitHub search bar search `outlier` usage in this repository (commits and code occurences), which would further the understanding of this function.
3. Let's try to improve the description of these functions with this newly acquired knowledge. 😀

Cheers,

**Comment by j143** [comment]
@wizard-420 - above comment!

File: dev/docs/builtins-reference.md (hunk 1 above)
Comment (wizard-420) [reply dev/docs/builtins-reference.md:295]: Actually I was not getting the correct format for outlier so I just replaced it. But don't worryi will add slicefinder again.

**Comment by wizard-420** [comment]
> Now, in a new PR add the documentation for `outliersd`.
> 
>     1. After which, you are aware of the outlier and it's flavours study a bit deeper and
> 
>     2. And in the GitHub search bar search `outlier` usage in this repository (commits and code occurences), which would further the understanding of this function.
> 
>     3. Let's try to improve the description of these functions with this newly acquired knowledge. grinning
> 
> 
> Cheers,

Finished with outliersd. On to the next task

**Comment by wizard-420** [comment]
I have uploaded the revised docs with correct description. please review it.

File: dev/docs/builtins-reference.md [hunk 2]
```diff
@@ -291,34 +291,95 @@ y = X %*% rand(rows=ncol(X), 1)
...
+```r
+X = rand (rows=10,cols=10)
+opposite = 1
+Y = outlier(X = X, opposite = opposite)
+Z = outlierByIQR(X=Y,k=1.5,repairMethod=0,max_iterations=3,verbose=1)
+print("\n"+toString(Z))
+`
+###outlierBySd - function
+Builtin function for detecting and repairing outliers using standard deviation
+
```
Comment (j143) [inline dev/docs/builtins-reference.md:361]: This is PR related to `OutlierIQR`, but we are having `OutlierBySd` changes. 🍰

Comment (j143) [reply dev/docs/builtins-reference.md:361]: @wizard-420

Comment (wizard-420) [reply dev/docs/builtins-reference.md]: thanks for your help.

**Comment by j143** [comment]
@wizard-420 - I made changes in this PR such that only `OutlierByIQR` related changes are available. :)

The similar thing can be done in other `outlier` related PR.
Also, they say `google search` is a developer's best friend!

**Comment by j143** [comment]
Closing this due to inactivity. Please reopen if needed. Thanks for contributing.

If you would like to contribute to the project 
1. Browse https://issues.apache.org/jira/projects/SYSTEMML/issues
2. or start a discussion on mailing list at dev@systemds.apache.org

**Pull request closed without merging**